mjsip-util/src/main/java/org/mjsip/config/OptionParser.java

```diff
@@ -71,13 +72,18 @@ public static MetaConfig parseOptions(String[] args, String defaultConfigFile, O
 			} catch (CmdLineException ex) {
 				// Happens, when required configurations are not given on the command line.
 				problem = ex;
+				String message = ex.getMessage();
+				invalidValueGiven = message != null && message.contains(" is not a valid value for ");
+				if (invalidValueGiven) {
+					LOG.warn("{}. Reading from environment/command line aborted", message);
+				}
 			}
 			
 			String argFile = metaConfig.configFile;
 			
 			File file;
 			if (argFile != null && !argFile.isBlank()) {
-				if ("none".equals(argFile.toLowerCase())) {
+				if ("none".equalsIgnoreCase(argFile)) {
```

Comment:
Gute Idee - wird das an anderen Stellen der Konfigurationsverarbeitung eigentlich auch so gemacht?

Reply:
Aktuell wird in phoneblock `recordings` noch nur mit equals geprüft und die Methode `getConfigFile` aus `MetaConfig` ist auch noch nicht angepasst.